modules/tracking/include/opencv2/tracking/tracker.hpp

```diff
@@ -1297,9 +1297,9 @@ class CV_EXPORTS_W TrackerGOTURN : public Tracker
   /** @brief Constructor
   @param parameters GOTURN parameters TrackerGOTURN::Params
   */
-  static Ptr<TrackerGOTURN> create(const TrackerGOTURN::Params &parameters);
+  static Ptr<TrackerGOTURN> create(const TrackerGOTURN::Params &parameters, const String modelTxt = "goturn.prototxt", const String modelBin = "goturn.caffemodel");
```

Comment:
Why not to include it inside the `parameters`?

Reply:
@alalek said:
"It is better to add extra parameters (into .create() or dedicated .set/.get calls) to specify location of model files with defaults "goturn.caffemodel" / "goturn.prototxt" (from the current directory)."

Reply:
Use `TrackerGOTURN::Params` in this case.

(previous comment was about `::create()` call without any visible parameters, so it was wrong)